### PR TITLE
LPS-72555

### PR DESCRIPTION
**Pull request by holatuwol**
Sending https://github.com/liferay/com-liferay-journal/pull/324 without the changes for LPS-72507, since equivalent changes have been committed:

* https://github.com/liferay/com-liferay-journal/commit/8cf0a3445309637420cc4bd0c647eec45387261d
* https://github.com/liferay/com-liferay-journal/commit/614086ab789746d0f4b222b6f7e59012b98d1869
* https://github.com/liferay/com-liferay-journal/pull/326

### CODE REVIEW

**Comment by ealonso** [comment]
ci:retest

File: journal-content-web/src/main/java/com/liferay/journal/content/web/internal/display/context/JournalContentDisplayContext.java [hunk 1]
```diff
@@ -598,8 +599,15 @@ public String getURLEdit() {
...
+			PortletRequest portletRequest = _portletRequest;
+
+			while (portletRequest instanceof PortletRequestWrapper) {
+				portletRequest =
+					((PortletRequestWrapper)portletRequest).getRequest();
+			}
+
 			PortletURL portletURL = latestArticleAssetRenderer.getURLEdit(
-				(LiferayPortletRequest)_portletRequest, null,
+				(LiferayPortletRequest)portletRequest, null,
```
Comment (ealonso) [inline journal-content-web/src/main/java/com/liferay/journal/content/web/internal/display/context/JournalContentDisplayContext.java]: Hey @shuyangzhou,

I'm not completely sure about this changes, Could you please help me taking a look at it?

Thanks!!

Comment (Preston-Crary) [inline journal-content-web/src/main/java/com/liferay/journal/content/web/internal/display/context/JournalContentDisplayContext.java]: We should use PortalImpl.getLiferayPortletRequest(PortletRequest), I'm going to send a pull to remove the other ones like this as part of https://issues.liferay.com/browse/LPS-65930.

**Comment by ealonso** [comment]
Sent to @Preston-Crary because I need some help with the review https://github.com/Preston-Crary/com-liferay-journal/pull/2

**Pull request closed without merging**